examples/typescript/package.json

```
@@ -0,0 +1,18 @@
+{
+    "name": "sdkexamples",
+    "version": "1.0.0",
+    "description": "Looker SDK examples in Typescript/Javascript",
+    "main": "index.ts",
+    "repository": "https://github.com/looker-open-source/sdk-codegen/tree/master/examples/typescript",
+    "author": "Looker",
+    "license": "MIT",
+    "private": false,
```

```suggestion
    "private": true,
```

We won't be publishing the sdk examples package.